example/20-network.yaml

```diff
@@ -46,11 +46,15 @@ spec:
   providerConfig:
     apiVersion: calico.networking.extensions.gardener.cloud/v1alpha1
     kind: NetworkConfig
-    backend: bird
-    ipam:
-      type: host-local
-      cidr: usePodCIDR
+    overlay:
+      enabled: false
+    snatToUpstreamDNS:
+      enabled: true
+#   ebpfDataplane:
+#     enabled: true
 #   ipAutodetectionMethod: interface=eth0
```

Comment:
Can you please also remove this entry? It seems deprecated and should not be included in the example anymore.